app/controllers/order_management/order_details_controller.rb

```diff
@@ -111,4 +113,9 @@ def edit_disabled?
     @order_detail.in_open_journal? || @order_detail.reconciled?
   end
 
+  def init_order_detail
```

Comment:
@LeticiaErrandonea why is this change needed?

Reply:
@giladshanan because the order and order detail may not be from the `current_facility` but from some other facility (for cross core orders)